content/logs/parsing.md

```diff
@@ -93,20 +93,20 @@ Here is the list of all the filter natively implemented by Datadog:
 |`keyvalue([separatorStr[, characterWhiteList [, quotingStr]])` |extracts key value pattern and returns a JSON object. [More info](#key-value) |
 |`scale(factor)` | multiplies the expected numerical value by the provided factor.|
 |`array([[openCloseStr, ] separator][, subRuleOrFilter)` | parses a string sequence of tokens and returns it as an array.|
-|`url`|parses an url and returns all the tokenized members (domain, query params, port, etc) in a JSON object. [More info](/logs/processing/#url-parser)|
+|`url`|parses a url and returns all the tokenized members (domain, query params, port, etc) in a JSON object. [More info](/logs/processing/#url-parser)|
 
 ## Examples
-Let’s see some examples to understand better how this works.
+Let’s see some examples to better understand how this works.
```

Comment:
Below you will find some examples demonstrating how to use parsers.

Reply:
👍